tools/demarkus-broker/internal/broker/templates/device_form.html

```
@@ -0,0 +1,27 @@
+  form { display: flex; flex-direction: column; gap: 0.75rem; margin-top: 1.5rem; }
+  input[type=text] { font: inherit; font-family: ui-monospace, monospace; font-size: 1.25rem; padding: 0.75rem; letter-spacing: 0.1em; text-transform: uppercase; border: 1px solid #ccc; border-radius: 4px; }
+  button { font: inherit; padding: 0.75rem; background: #111; color: #fff; border: 0; border-radius: 4px; cursor: pointer; }
+  button:hover { background: #333; }
+  .error { color: #b00020; margin-top: 0.5rem; }
+</style>
+</head>
+<body>
+<h1>Connect device</h1>
+<p>Enter the code shown by your terminal or editor to finish joining this demarkus broker.</p>
+<form method="post" action="/device">
+  <input type="text" name="user_code" value="{{.PrefilledUserCode}}" placeholder="XXXX-XXXX" autofocus autocomplete="off" autocapitalize="characters" spellcheck="false" required>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)